great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -1,11 +1,15 @@
 from great_expectations.data_context.data_context.abstract_data_context import (
     AbstractDataContext,
 )
+from great_expectations.data_context.types.data_context_variables import (
+    CloudDataContextVariables,
+)
 
 
 class CloudDataContext(AbstractDataContext):
     """
     Subclass of AbstractDataContext that contains functionality necessary to hydrate state from cloud
     """
 
-    pass
+    def _init_variables(self) -> CloudDataContextVariables:
+        raise NotImplementedError
```

Comment:
Subsequent PR will bring these to life (waiting on @Shinnnyshinshin's DataContext refactor to ensure we have the variables we need)